### PR TITLE
feat(carousel):uses `n` as CSS vars prefix

### PR DESCRIPTION
**Pull request by doom-9**

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### Fixes
+
+- `n-carousel` uses `n` as CSS vars prefix.
```
Comment (Sepush) [inline CHANGELOG.en-US.md:5]: feat

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### Fixes
+
+- `n-carousel` 的 CSS 变量使用 `n` 作为前缀
```
Comment (Sepush) [inline CHANGELOG.zh-CN.md:5]: feat

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### Feats
```
Comment (Talljack) [inline CHANGELOG.en-US.md:3]: Pending

Feats

格式问题　下同

**Comment by Sepush** [comment]
hey you should make the commit be clean

**Pull request merged**